pallets/handles/src/handles-utils/src/validator.rs

```diff
@@ -0,0 +1,95 @@
+//! # Handle Validator
+//!
+//! `handle_validator` provides a `HandleValidator` struct to determine the validity of a given
+//! user handle.
+extern crate alloc;
+use alloc::vec::Vec;
+
+/// A validator for user handles.
+///
+/// Given a String user handle as input, determines if the handle is valid.
+pub struct HandleValidator<'string_lifetime> {
```

Comment:
I do not think we need this struct. We are not passing any data from outside. Maybe we move those things to constants at the top of the file?